schemas/schema.purchaseOrders.js

```diff
@@ -0,0 +1,22 @@
+const schema = require("mongoose").Schema;
+
+const patientSchema = {
+	_id: { type: String, required: true },
+	productId: { type: String, required: true, },
+	name: { type: String, required: true, unique: true },
+	unitOfMeasure: { type: String, required: true },
+	unitPrice: { type: Number, required: true },
+	requestedQuantity: { type: Number, required: true },
+	receivedQuantity: { type: Number, required: true },
+	isReceived: { type: Boolean, required: true },
+	orderStatus: { type: String, enum: ["Ordered", "Not ordered", "Pending"] },
```

Comment:
Purchase Order or PO is against a vendor. So you should also have a section to capture vendor information